app/classes/document_query.rb

```diff
@@ -189,10 +189,12 @@ def build_search_query
                             end
                           end
 
-                          must do
-                            bool do
-                              doc_query.full_text_fields.values.each do |field|
-                                should { common({ field => doc_query.common_terms_hash }) }
+                          unless doc_query.query.match(/".*"/)
+                            must do
+                              bool do
+                                doc_query.full_text_fields.values.each do |field|
+                                  should { common({ field => doc_query.common_terms_hash }) }
```

Comment:
Note: the Elasticsearch DSL is quite finicky, so that hash rocket needs to remain a hash rocket.